WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsAndTopPerformersViewController.swift

```diff
@@ -431,6 +440,47 @@ private extension StoreStatsAndTopPerformersViewController {
         stackView.addArrangedSubview(separator)
         return stackView
     }
+
+    func startCustomRangeTabCreation() {
+        guard let navigationController else { return }
+        customRangeCoordinator = CustomRangeTabCreationCoordinator(
+            navigationController: navigationController,
+            onDateRangeSelected: { [weak self] start, end in
+                self?.createCustomRangeTab(start, end)
+            }
+        )
+        customRangeCoordinator?.start()
+    }
+
+    func createCustomRangeTab(_ start: Date, _ end: Date) {
+        let currentDate = Date()
+
+        // TODO: 11935 Add the correct data fetching and displaying based on custom range.
+        let customRangeVC = StoreStatsAndTopPerformersPeriodViewController(siteID: siteID,
+                                                                           timeRange: .custom(from: start, to: end),
+                                                                           currentDate: currentDate,
+                                                                           canDisplayInAppFeedbackCard: true,
```

Comment:
I think this should be `false` as I see that for other tabs we set this as true only when `timeRange == .today`.

Reply:
Nice observation. Handled in https://github.com/woocommerce/woocommerce-ios/pull/11977/commits/bac4c2e06f2bcbc028aefea59a8008d8395a50b2